Convex_hull_3/examples/Convex_hull_3/quickhull_3_SM.cpp

```diff
@@ -16,7 +15,7 @@ int main(int argc, char* argv[])
 {
   std::ifstream in( (argc>1)? argv[1] : "data/star.off");
   
-  Polyhedron_3 poly;
+  Surface_mesh poly;
```

Comment:
The point was the example is not testing a `Polyhedron` or a `Surface_mesh` but is using a graph.